dcat/index.html

```diff
@@ -1482,7 +1494,427 @@ <h4>Property: is referenced by</h4>
+</tr>
+<tr>
+<td class="prop">Equivalent property:</td>
+<td><a data-cite="?PAV#d4e395"><code title="http://purl.org/pav/hasVersion">pav:hasVersion</code></a></td>
+</tr>
+<tr>
+<td class="prop">Sub-property of:</td>
+<td><a data-cite="?DCTERMS#http://purl.org/dc/terms/hasVersion"><code title="http://purl.org/dc/terms/hasVersion">dct:hasVersion</code></a></td>
+</tr>
+<td class="prop">Sub-property of:</td>
+<td><a data-cite="?PROV-O#generalizationOf"><code title="http://www.w3.org/ns/prov#generalizationOf">prov:generalizationOf</code></a></td>
+</tr>
+<tr>
+<td class="prop">Usage note:</td>
+<td>
+<p>This property is intended for relating a non-versioned or abstract resource to several versioned resources, e.g. snapshots [[?PAV]].</p>
+<p>The notion of version used by this property is limited to versions resulting from revisions occurring to a resource as part of its life-cycle. Therefore, its semantics is more specific than its super-property <a data-cite="?DCTERMS#http://purl.org/dc/terms/hasVersion"><code title="http://purl.org/dc/terms/hasVersion">dct:hasVersion</code></a>, which makes use of a broader notion of version, including editions and adaptations.</p>
+</td>
+</tr>
+</tbody>
+</table>
+
+<p>For guidance on the use of this property, see <a href="#version-history"></a>.</p>
+
+</section>
+
+<section id="Property:resource_is_version_of">
+
+<h4>Property: is version of</h4>
+
+<aside class="note">
+<p>Property added in DCAT 3.</p>
+</aside>
+
+<table class="definition">
+<thead>
+<tr>
+<th>RDF Property:</th>
+<th><a href="http://www.w3.org/ns/dcat#isVersionOf">dcat:isVersionOf</a></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td class="prop">Definition:</td>
+<td>This resource is a version of a non-versioned or abstract resource.</td>
+</tr>
+<tr>
+<td class="prop">Inverse property of:</td>
+<td><a href="#Property:resource_has_version"><code title="http://www.w3.org/dcat#hasVersion">dcat:hasVersion</code></a></td>
+</tr>
+<tr>
+<td class="prop">Sub-property of:</td>
+<td><a data-cite="?DCTERMS#http://purl.org/dc/terms/isVersionOf"><code title="http://purl.org/dc/terms/isVersionOf">dct:isVersionOf</code></a></td>
+</tr>
+<td class="prop">Sub-property of:</td>
+<td><a data-cite="?PROV-O#specializationOf"><code title="http://www.w3.org/ns/prov#specializationOf">prov:specializationOf</code></a></td>
+</tr>
+<tr>
+<td class="prop">Usage note:</td>
+<td>
+<p>This property is intended for relating a versioned resource to a non-versioned or abstract resource.</p>
+<p>The notion of version used by this property is limited to versions resulting from revisions occurring to a resource as part of its life-cycle. Therefore, its semantics is more specific than its super-property <a data-cite="?DCTERMS#http://purl.org/dc/terms/isVersionOf"><code title="http://purl.org/dc/terms/isVersionOf">dct:isVersionOf</code></a>, which makes use of a broader notion of version, including editions and adaptations.</p>
```

Comment:
```suggestion
<p>The notion of version used by this property is limited to versions resulting from revisions occurring to a resource as part of its life-cycle. Therefore, its semantics is more specific than its super-property <a href="http://purl.org/dc/terms/isVersionOf"><code title="http://purl.org/dc/terms/isVersionOf">dct:isVersionOf</code></a>, which makes use of a broader notion of version, including editions and adaptations.</p>
```

Reply:
Proposal not clear. The original markup should work, and point to the relevant part of the DCTERMS spec. Am I missing something?

Reply:
the original markup does not work,  it is resolved in "https://dublincore.org/documents/dcmi-terms//purl.org/dc/terms/isVersionOf" instead of "https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/isVersionOf". 
And similarly for the other links included in this dct in pull.

Reply:
Thanks, @riccardoAlbertoni . I'll check it out.

Reply:
@riccardoAlbertoni , you were right. It seems that ReSpec was messing up the link. I fixed the issue throughout the document via commit https://github.com/w3c/dxwg/pull/1295/commits/7f06964f279a1b1e62babd8f8242befc0bbe80b2

Reply:
@riccardoAlbertoni , can we mark as resolved these change proposals?